docs/EN/Configuration/OmnipodDASH.md

```diff
@@ -1,91 +1,567 @@
 
-# About this documentation
+About this documentation
```

Comment:
We are keeping this while working on the docs or do we plan to keep it forever?
If it's temporary, maybe we could move it in a comment in the PR?

Reply:
Yep we can move it to comments